autogen/oai/client.py

```diff
@@ -488,6 +497,8 @@ def _completions_create(self, client: OpenAI, params: Dict[str, Any]) -> ChatCom
 
                         # If content is present, print it to the terminal and update response variables
                         if content is not None:
+                            if self.websocket is not None:
+                                self.websocket.send(content)
```

Comment:
Can this be made asynchronous?

Reply:
@ekzhu I did not plan for this to be asynchronous, but you are right, I should include that functionality too.